frontend/assets/javascripts/src/modules/form/options.js

```diff
@@ -0,0 +1,199 @@
+        });
+    }
+
+    function renderCardDetailsNotes(period) {
+        toArray(CARD_NOTE_CURRENCIES_ELS).forEach(function (el) {
+            toggleCurrency(el);
+        });
+
+        var attr = 'data-' + period;
+        var paymentTakenEl = $(dataCurrencySel(checkoutForm.currency, CARD_NOTE_PRICING_CHARGE), CARD_DETAILS_NOTE_EL)[0];
+        var periodEl = $(dataCurrencySel(checkoutForm.currency, CARD_NOTE_PERIOD), CARD_DETAILS_NOTE_EL)[0];
+
+        paymentTakenEl.innerHTML = paymentTakenEl.getAttribute(attr);
+        periodEl.innerHTML = periodEl.getAttribute(attr);
+    }
+
+    function renderSubmitButton() {
+        toArray($(DATA_CURRENCY_SEL, SUBMIT_BTN_EL)).forEach(function (el) {
+            var elCurrency = $(el).attr('data-currency');
+            var elBillingPeriod = $(el).attr('data-billing-period');
+
+            if (checkoutForm.currency === elCurrency && checkoutForm.billingPeriod === elBillingPeriod) {
+                el.classList.remove('is-hidden');
+            } else {
+                el.classList.add('is-hidden');
+            }
+        });
+    }
+
+    function setBillingPeriod(el) {
+        toArray(el.querySelectorAll('input[type="radio"]')).forEach(function (inputEl) {
+            var elBillingPeriod = inputEl.value;
+            var elCurrency = inputEl.getAttribute(CURRENCY_ATTR);
+            if (elBillingPeriod === checkoutForm.billingPeriod && elCurrency === checkoutForm.currency) {
+                inputEl.checked = true;
+            }
+            else {
+                inputEl.checked = false;
+            }
+        });
+    }
+
+    var init = function () {
+        checkoutForm.deliveryCountry = checkoutForm.defaultCountry;
+
+        if (PAYMENT_OPTIONS_CONTAINER_EL && checkoutForm) {
+            checkoutForm.showBillingAddress = false;
+            checkoutForm.billingCountry = checkoutForm.deliveryCountry;
+            addListeners();
+            renderPrices();
+        }
+
+        if (FRIEND_FORM_EL && checkoutForm) {
+            selectCountry(DELIVERY_COUNTRY_EL, checkoutForm.deliveryCountry);
+        }
+    };
+
+    var addListeners = function () {
+        bean.on(PAYMENT_OPTIONS_CONTAINER_EL, 'click', 'input', function (e) {
+            var input = e.target;
+            checkoutForm.billingPeriod = $(input).val();
+            renderPrices();
+        });
+
+        bean.on(DELIVERY_COUNTRY_EL, 'change', function(e) {
+            var input = e.target;
+            var selectedCountry = $(input).val();
+
+            checkoutForm.deliveryCountry = selectedCountry;
+            if (!checkoutForm.showBillingAddress) {
+                checkoutForm.billingCountry = selectedCountry;
+            }
+
+            renderPrices();
+        });
+
+        bean.on(BILLING_COUNTRY_EL, 'change', function(e) {
+            var input = e.target;
+            var selectedCountry = $(input).val();
+            var selectedEl = toArray($('option', input)).filter(function (el) {
```

Comment:
Actually I think that's just a [jQuery thing](https://api.jquery.com/selected-selector/)... of course it can't be that simple